helm-chart/openj9-jitserver-chart/Chart.yaml

```diff
@@ -0,0 +1,47 @@
+###############################################################################
+# Copyright (c) 2020, 2021 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+apiVersion: v2
+name: openj9-jitserver-chart
+description: |-
+  Eclipse OpenJ9 JITServer Helm Chart. 
+
+  License
+  By installing this product you accept the GNU General Public License, 
+  version 2 with the GNU Classpath Exception [1] and GNU General Public 
+  License, version 2 with the OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
```

Comment:
```suggestion

This chart is made available under the terms of the Eclipse Public License 2.0 which accompanies this
distribution and is available at https://www.eclipse.org/legal/epl-2.0/ or the Apache License, Version 2.0 which accompanies this distribution and is available at https://www.apache.org/licenses/LICENSE-2.0.

The JDK binaries installed by this chart are licensed under the GPLv2+CE.
```

Reply:
There are two things here - calling out the license of the chart and the binaries it installs should be done separately

Reply:
Thanks. I have updated the license explanation in the latest commit.